geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/pubsub/PubSub.java

```diff
@@ -31,10 +31,8 @@
   /**
    * Publish a message on a channel
    *
-   *
    * @param channel to publish to
    * @param message to publish
-   * @return the number of messages published
    */
   long publish(RegionProvider regionProvider, byte[] channel, byte[] message);
```

Comment:
since publish still returns a long it seems like you should still have a comment describing what is returned. How about "return an estimate of the number of messages published by this call". It might also be worth enhancing this comment to say that this call may return before the actual publish has been done. Something about it being async.

Reply:
Docs updated with better comments.